jwst/assign_wcs/util.py

```diff
@@ -100,25 +100,40 @@ def compute_scale(wcs: WCS, fiducial: Union[tuple, np.ndarray]) -> float:
         Reference WCS object from which to compute a scaling factor.
 
     fiducial : tuple
-        Input fiducial of (RA, DEC) used in calculating reference points.
+        Input fiducial of (RA, DEC) or (RA, DEC, Wavelength) used in calculating reference points.
+
+    spec : bool
```

Comment:
This name is a bit vague - `spec` can mean specification or spectral.  I would make it `spectral` or even `spectral_wcs`.

Or maybe the function doesn't need this kwarg and can figure out internally if it is a spectral WCS?  I.e. the output frame will have both spatial and spectral components.

Reply:
Ah, so something like: 

```python
if 'SPECTRAL' in wcs.output_frame.axes_type:
    ...  # do the things for spectral WCSs
```